docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:buster
+ENV DEBIAN_VERSION=buster-20200327
+ENV ODGI_COMMIT=dc76da391943a6bd6a7784e42f946dc478b865a5
+
+WORKDIR "/opt/"
+RUN DEBIAN_FRONTEND=noninteractive \
+    && apt-get update -y \
+    && apt-get install -y --no-install-recommends \
+      build-essential \
+      ca-certificates \
+      cmake \
+      git \
+      python3-dev \
+    #  python3-setuptools \
+      zlib1g-dev \
+      libgomp1 \
+    && rm -fr /var/lib/apt/lists/* \
+    && update-ca-certificates \
+    && git clone --shallow-submodules --recursive https://github.com/vgteam/odgi.git \
+    && cd odgi \
+    && git checkout "${ODGI_COMMIT}" \
+    && cmake -DBUILD_STATIC=0 -H. -Bbuild \
+    && cmake --build build -- -j $(nproc) \
+    && rm -rf deps \
+    && rm -rf .git \
+    && rm -rf build \
+    && rm -rf lib/*.a \
+    && ls -s bin/odgi /usr/bin \
+    && apt-get remove -y --autoremove  build-essential cmake git \
+    && apt-get clean \
+    && apt-get purge \
+    && rm -rf /var/lib/apt/lists/* /tmp/*
```

Comment:
If you use build stages, then all this deleting isn't necessary

Here's my minimal image for odgi that demonstrates this technique in 201 MiB
```
FROM debian:stable-slim
WORKDIR /usr/src/app
RUN apt-get update && apt-get install -y git bash cmake make g++ python-dev && rm -rf /var/lib/apt/lists/*
RUN git clone --recursive https://github.com/vgteam/odgi.git
RUN cd odgi && cmake -H. -Bbuild && cmake --build build -- -j $(nproc) && cd build && make install
FROM debian:stable-slim
RUN apt-get update && apt-get install -y libgomp1 && rm -rf /var/lib/apt/lists/*
COPY --from=0 /usr/local/bin/odgi /usr/local/bin/
```

Reply:
@mr-c thanks, that was a very helpful hint about build stages.

I made two post build stages, one as above.
The second with the minimal python3 install for downstream users using python3.

@subwaystation do you need both the binary and the python3 libs?

Reply:
Until now, we would only need the binary for our workflow. I don't expect we will need the bindings in the future. Fingers crossed.

But, maybe @Flavia95 would have interest in a docker including Python3 libs?

Reply:
Wow, thanks @subwaystation.
That would be great for me. It would make my job much easier.